.github/workflows/precommit.yml

```diff
@@ -515,9 +515,6 @@ jobs:
         with:
           name: overall_cov_xml
           path: .tox/coverage.xml
-      - uses: codecov/codecov-action@v1
```

Comment:
Is there a point to keep the `Combine coverages` and  `Publish html coverage` jobs?

Reply:
You can remove the whole `combine-coverage:` job